templates/guides/opting_into_beta_resources.md.tmpl

```diff
@@ -0,0 +1,44 @@
+---
+page_title: "Configuring Beta Resources in the STACKIT Terraform Provider"
+---
+# Configuring Beta Resources in the STACKIT Terraform Provider
+
+### Overview
+
+This guide explains how to opt into beta resources within the STACKIT Terraform provider. Beta resources are new services and features from the STACKIT cloud that are still in development and do not yet have a stable API. 
```

Comment:
```suggestion
This guide explains how to opt into beta resources within the STACKIT Terraform provider. Beta resources are new services and features from the STACKIT cloud that are still in development and might not yet have a stable API. 
```

Reply:
This one is missing, right?

Reply:
Ah, I applied the previous suggestion after and it removed the change